### PR TITLE
Portlet tests rewritten.

### PR DESCRIPTION
**Pull request by rhopp**

### CODE REVIEW

**Comment by rhopp** [comment]
Nightly build up and running: https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Devstudio_integration_tests/job/portlet.nightly/4/

File: tests/org.jboss.tools.portlet.bot.test/pom.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		</resources>
+
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-surefire-plugin</artifactId>
+				<configuration>
+					<testSuite>org.jboss.tools.portlet.bot.test</testSuite>
+					<testClass>${test.class}</testClass>
+					<explodedBundles>
```
Comment (rawagner) [inline tests/org.jboss.tools.portlet.bot.test/pom.xml]: this config is already defined in parent

File: tests/org.jboss.tools.portlet.bot.test/pom.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+						<bundle>org.mozilla.xulrunner.cocoa.macosx</bundle>
+						<bundle>org.mozilla.xulrunner.gtk.linux.x86</bundle>
+						<bundle>org.mozilla.xulrunner.gtk.linux.x86_64</bundle>
+						<bundle>org.mozilla.xulrunner.win32.win32.x86</bundle>
+					</explodedBundles>
+					<dependencies combine.children="append">
```
Comment (rawagner) [inline tests/org.jboss.tools.portlet.bot.test/pom.xml]: is this part needed ?

**Comment by rhopp** [comment]
@rawagner done. I left org.jboss.tools.portlet.feature.feature.group in there because it's not part of default installation.

**Pull request merged**